src/components/button-toggle/button-toggle.component.js

```diff
@@ -48,11 +54,18 @@ const ButtonToggle = (props) => {
 
   return (
     <StyledButtonToggle
-      data-component="button-toggle"
+      aria-checked={checked}
```

Comment:
Shouldn't `aria-checked` be applied to the `ButtonToggleInput` ?

Reply:
the input is hidden, so it should be ignored by readers

Reply:
looks like the VoiceOver utility is reading that input anyway, so this aria prop is redundant